EDSEditorGUI/DeviceInfoView.cs

```diff
@@ -137,11 +139,13 @@ private void update_devfile_info()
             try
             {
                 eds.di.ProductName = textBox_productname.Text;
-                eds.di.ProductNumber = textBox_productnumber.Text;
+                eds.di.ProductNumber = EDSsharp.U32Parse(textBox_productnumber.Text);
                 eds.di.VendorName = textBox_vendorname.Text;
-                eds.di.VendorNumber = textBox_vendornumber.Text;
+                eds.di.VendorNumber = EDSsharp.U32Parse(textBox_vendornumber.Text);
+                eds.di.RevisionNumber = EDSsharp.U32Parse(textBox_revisionnumber.Text);
```

Comment:
just curious, what is the reason that some are hex and some are not?
For instance i would imagine revision would be non-hex.